### DISCUSSION TITLE
magit-todos

### DISCUSSION THREAD

**Discussion by bhw-foss** [syl20bnr/spacemacs#16293] [Ideas]
Good morning,

I saw this on Sacha's weekly Emacs newsletter: https://github.com/alphapapa/magit-todos

I think this is useful enough to add to the magit layer. Thoughts? Opinions?

**Comment by alexey0308** [comment]
Hi @BenedictHW,

you are able to use it already by specifying

```
  (setq-default dotspacemacs-configuration-layers
                '((git :variables git-enable-magit-todos-plugin t)))

```
in you config.

Please take a look at the `git` layer [documentation](https://develop.spacemacs.org/layers/+source-control/git/README.html#magit-todos).

**Comment by bhw-foss** [comment]
D'oh! My bad haha, no wonder placing "magit-todo" in
`dotspacemacs-additional-packages' didn't do anything.

Cheers,

**Reply by bhw-foss** [reply]
Thank you Alexey
